librz/core/cmd/cmd_debug.c

```diff
@@ -2697,33 +2453,27 @@ RZ_IPI int rz_cmd_debug(void *data, const char *input) {
 			}
 		} break;
 		case '-': // "dd-"
-			// close file
-			{
-				int fd = atoi(input + 2);
-				// rz_core_cmdf (core, "dxs close %d", (int)rz_num_math ( core->num, input + 2));
-				RzBuffer *buf = rz_core_syscallf(core, "close", "%d", fd);
-				consumeBuffer(buf, "dx ", "Cannot close");
-			}
-			break;
+			  // close file
```

Comment:
You can put comments and `break` inside braces, then it will not harm the indentation. But it's low-priority anyway, since this part will be removed in favor of the rzshell handlers anyway.